config/loggerConfig.js.example

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  logLevel: 'debug',  // options:
```

Comment:
It would be nice to show other available options and just comment them out. Also explain what they do. I don't know what `logLevel` means, what are the other possible values besides `debug`?